content/issues/1/data-beyond-vision/index.md

```diff
@@ -209,7 +209,7 @@ The bar chart conveys a sense of certitude and exactness that does not reflect t
 ### Next Steps
 The current version uses different shapes for the two variables, but adding textures would make the model even more tactile. Simple 3D printed labels with text and braille[^28] have been added for display alongside the piece, but they could be incorporated directly on the model, and refined to provide a scale for the axes. The 3D printed objects could also be augmented with other media: lights or sound could convey the intensity of borrowing activity, or threads connecting months to represent the number of subscription renewals and convey a sense of continuity. The Python code used to create these models could be generalized for reuse, and eventually made available as a Blender plugin.[^29] New shapes or approaches could be used to leverage innovations in 3D modeling, such as generative design, to create objects that are more inviting to touch and even more distinct from 2D data visualizations. 3D models could be revised for fabrication with CNC machines to create objects out of wood instead of plastic, which could make them more inviting to touch.
 
-[Make one >](https://startwords.cdh.princeton.edu/issues/1/print-a-model-of-the-shakespeare-and-company-lending-library-membership/)
+[Make one >](/issues/1/modeling-howto/)
```

Comment:
are we for sure good with relative links for these? I used absolute URLs because I wanted the regex that expands them in the text-only output to create a link that would actually work out of context, but that might not be that important.

Reply:
I wondered why you were using absolute links! It def. makes it harder to test the links on a different instance.

Relative links in the text versions sounds like an editorial decision to me, defer to @gwijthoff . If we care about having full links, I'd much rather see a regex solution that adds the base url vs hardcoding it to the production site.